src/list/types.js

```diff
@@ -44,10 +44,10 @@ export type LabelPropsT = {|
 export type MenuAdapterPropsT = {
   ...PropsT,
   // eslint-disable-next-line flowtype/no-weak-types
-  item: any,
-  onMouseEnter: MouseEvent => mixed,
-  onClick: MouseEvent => mixed,
-  $size: string,
-  $isHighlighted: boolean,
-  $disabled: boolean,
+  item?: any,
```

Comment:
why relax this type?

Reply:
Don't want to restrict the menu item type since with custom renderers they may want to include various additional data to be reused on render.